src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#header">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="header__logo__img"
```

Comment:
According to BEM, an element cannot contain another element directly. Only blocks have elements. Therefore, `header__logo-img` is the correct class name, where `header` is the block and `logo-img `is the element.
```suggestion
            class="header__logo-img"
```